scripts/packages/packager/local-entrypoint.sh

```diff
@@ -24,17 +31,21 @@ chmod +x /staging/usr/local/etc/rc.d/nginx-agent
 git config --global --add safe.directory /nginx-agent
 VERSION="$(git describe --match 'v[0-9]*' --abbrev=0 | tr -d 'v')-SNAPSHOT-$(git rev-parse --short HEAD)" envsubst < scripts/packages/manifest > /staging/+MANIFEST
 
-pkg -o ABI="FreeBSD:13:amd64" create --format txz  \
-    -m /staging \
-    -r /staging \
-    -p /staging/plist \
-    -o ./build/; \
-
-# Creating symbolic link from txz to pkg. In older versions of pkg the extension would represent the format of the file 
-# but since version 1.17.0 pkg will now always create a file with the extesion pkg no matter what the format is. 
-# See 1.17.0 release notes for more info: https://cgit.freebsd.org/ports/commit/?id=e497a16a286972bfcab908209b11ee6a13d99dc9
-cd build/
-ln -s nginx-agent-"$(git describe --match 'v[0-9]*' --abbrev=0 | tr -d 'v')"-SNAPSHOT-"$(git rev-parse --short HEAD)".pkg nginx-agent-"$(git describe --match 'v[0-9]*' --abbrev=0 | tr -d 'v')"-SNAPSHOT-"$(git rev-parse --short HEAD)".txz 
-cd ..
+for freebsd_abi in $FREEBSD_DISTROS; do
```

Comment:
For local packages, we typically build one of each type e.g. deb, rpm, apk, etc. We don't build all possible versions. Could we remove the for loop and just build one package but this time using the new `ABIARCH` variable you have defined?
i.e.
```
pkg -o ABI="FreeBSD:13:${ABIARCH}" create --format txz \
....
```
That way we don't need to add the additional `cp` command in the ci.yml file.

Reply:
Makes sense. Will do shortly.

Reply:
(I did it that way to reduce a diff between "local" and "signed" scripts - just to explain the rationale.)